### PR TITLE
Updated documentation

### PR DESCRIPTION
**Pull request by VeronikaSolovei9**

### CODE REVIEW

File: docs/developers/add-new-bidder.md [hunk 1]
```diff
@@ -33,6 +33,13 @@ Bidder implementations are scattered throughout several files.
 
 Bidder implementations may assume that any params have already been validated against the defined json-schema.
 
+###Long form video support
+If bidder is going to support long form video make sure bidder returns:
+
+- single [IAB subcategory](https://iabtechlab.com/standards/content-taxonomy/) for each bid in `TypedBid.bid.Cat` field
```
Comment (guscarreon) [inline docs/developers/add-new-bidder.md]: Will this `TypeBid.bid.Cat` be a string representing one of the `Content Taxonomy v2 with Tiered Categories` found in the excel file downloadable from `https://iabtechlab.com/standards/content-taxonomy/`? Or will this field be an `int` representing the `UniqueID` found in said excel file? Maybe it is an object with both the `UniqueID` and `Tier1...4` categories. Can we be a little more specific?

Comment (VeronikaSolovei9) [reply docs/developers/add-new-bidder.md]: @guscarreon link updated, please review again

Comment (VeronikaSolovei9) [reply docs/developers/add-new-bidder.md]: @guscarreon thank you, I added a little more info, please review again. Sorry about this.

File: docs/developers/add-new-bidder.md [hunk 2]
```diff
@@ -33,6 +33,13 @@ Bidder implementations are scattered throughout several files.
 
 Bidder implementations may assume that any params have already been validated against the defined json-schema.
 
+###Long form video support
+If bidder is going to support long form video make sure bidder returns:
+
+- category in primary ad server format in `bid.bidVideo.PrimaryCategory`, string value, or single IAB category for each bid in `TypedBid.bid.Cat` field, singe string value in string list, please refer to this convenience link (not the final official definition): [IAB categories](https://adtagmacros.com/list-of-iab-categories-for-advertisement/)
```
Comment (guscarreon) [inline docs/developers/add-new-bidder.md]: Veronica, I believe there is a typo in the word "singe" I bet you meant "single". I find the sentence a little confusing, can we please rephrase it? Did you mean that the bidder will return a `string` with either the value found in `bid.bidVideo.PrimaryCategory` or the one in `TypedBid.bid.Cat`?

Comment (VeronikaSolovei9) [reply docs/developers/add-new-bidder.md]: @guscarreon  oh yes, sure, will fix and rephrase. Bidder must return one of those: `bid.bidVideo.PrimaryCategory`, string, or `TypedBid.bid.Cat` just first string in string array.

Comment (VeronikaSolovei9) [reply docs/developers/add-new-bidder.md]: Updated

File: docs/developers/add-new-bidder.md [hunk 3]
```diff
@@ -33,6 +33,16 @@ Bidder implementations are scattered throughout several files.
 
 Bidder implementations may assume that any params have already been validated against the defined json-schema.
 
+###Long form video support
+If bidder is going to support long form video make sure bidder returns:
+
+- category in `bid.bidVideo.PrimaryCategory` or `TypedBid.bid.Cat` field. <br>
+Category from `bid.bidVideo.PrimaryCategory` field should be a string value, translated to Primary ad server format.<br>
+Category from `TypedBid.bid.Cat` field should be a single string value in string array in IAB format.
+To learn more about IAB categories, please refer to this convenience link (not the final official definition): [IAB categories](https://adtagmacros.com/list-of-iab-categories-for-advertisement/)
```
Comment (guscarreon) [inline docs/developers/add-new-bidder.md]: 1) Can we fix the identation of these 3 sentences? Can they be in one paragraph?

Comment (guscarreon) [reply docs/developers/add-new-bidder.md]: 2) Can we please remove HTML `<br>` tags?
3) In the sentence "Category from `bid.bidVideo.PrimaryCategory` field should be a string value, translated to Primary ad server format", what is the "Primary add server format"?
4) The following sentence: "category in `bid.bidVideo.PrimaryCategory` or `TypedBid.bid.Cat` field. <br>" could be interpreted as: "Field, `bid.bidVideo.PrimaryCategory` can also be called `TypedBid.bid.Cat`". Can we rephrase?

File: docs/developers/add-new-bidder.md [hunk 4]
```diff
@@ -33,6 +33,14 @@ Bidder implementations are scattered throughout several files.
 
 Bidder implementations may assume that any params have already been validated against the defined json-schema.
 
+###Long form video support
```
Comment (mansinahar) [inline docs/developers/add-new-bidder.md]: I believe you need a space here after `###` for it to be correctly formatted. Without the space it doesn't show `Long form video support` as a heading but just normal text with `###` prepended to it.

**Comment by mansinahar** [comment]
@VeronikaSolovei9 Just a thought, what do you think about converting this information in a table rather? Something like the following:

```
| Field          |Type                           |Description                       
|----------------|-------------------------------|-----------------------------|
|bid.bidVideo.PrimaryCategory | string | Category for the bid. Should be able to be translated to Primary ad server format           
|TypedBid.bid.Cat | []string | Category for the bid. Should be an array with length 1 containing the value in IAB format. For more information about IAB Categories refer [this](https://adtagmacros.com/list-of-iab-categories-for-advertisement/).             
|TypedBid.BidVideo.Duration | int | AD duration in seconds
|TypedBid.bid.Price | float | Bid price
```

For the category information, I am not sure if they can specify only either of `bid.bidVideo.PrimaryCategory` or `TypedBid.bid.Cat` or both but whatever that is we can specify that in the description for those field also.

File: docs/developers/add-new-bidder.md (hunk 4 above)
Comment (VeronikaSolovei9) [reply docs/developers/add-new-bidder.md]: Updated!

**Comment by VeronikaSolovei9** [comment]
@mansinahar updated to table format with some modifications. Please review.

**Comment by VeronikaSolovei9** [comment]
Thank you, Mansi!

**Comment by VeronikaSolovei9** [comment]
Agree, thank you, guys!

**Pull request merged**